src/pages/dashboard/analysis/components/OfflineData.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card, Col, Row, Tabs } from 'antd';
+import { FormattedMessage, formatMessage } from 'umi-plugin-react/locale';
+import React from 'react';
+import { OfflineChartData, OfflineDataType } from '../data.d';
+
+import { TimelineChart, Pie } from './Charts';
+import NumberInfo from './NumberInfo';
+import styles from '../style.less';
+
+const CustomTab = ({
+  data,
+  currentTabKey: currentKey,
+}: {
+  data: OfflineDataType;
+  currentTabKey: string;
+}) => (
+  <Row gutter={8} style={{ width: 138, margin: '8px 0' }} type="flex">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 8](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)